tests/e2e/support/utils.js

```diff
@@ -234,7 +234,7 @@ export function getNumberOfLinks() {
     .then(({ graph }) => graph.getLinks().length);
 }
 
-export function assertDownloadedXMLMatchesExpected(xmlString) {
+export function assertDownloadedXmlContainsExpected(xmlString) {
```

Comment:
I renamed this to use `Xml` instead of `XML`, as other functions use that casing. I also changed `Matches` to `Contains`, as it better reflects what the method is doing.

Reply:
Aww. I used this in the Message End Event branch.